packages/tx/src/eip4844Transaction.ts

```diff
@@ -82,7 +84,10 @@ const validateBlobTransactionNetworkWrapper = (
  * - TransactionType: 3
  * - EIP: [EIP-4844](https://eips.ethereum.org/EIPS/eip-4844)
  */
-export class BlobEIP4844Transaction extends BaseTransaction<TransactionType.BlobEIP4844> {
+export class BlobEIP4844Transaction
+  extends BaseTransaction<TransactionType.BlobEIP4844>
+  implements EIP4844CompatibleTxInterface<TransactionType.BlobEIP4844>
```

Comment:
Here again (and for the other tx types too), I think we should remove.

This blurs again the capability concept.

Reply:
reomved